fe/fe-core/src/test/resources/sql/optimizer/rewrite/week-function-test.dat

```diff
@@ -0,0 +1,4920 @@
+0	1900-01-05 00:00:00	0	1900-12-27 00:00:00	51	2000-01-05 00:00:00	1	2000-12-27 00:00:00	52
+1	1900-01-05 00:00:00	1	1900-12-27 00:00:00	52	2000-01-05 00:00:00	1	2000-12-27 00:00:00	52
```

Comment:
do we really need this so large test data set in UT?

Reply:
> Why not? Week implementation is very complicated, why not prepare a large test case?

@dirtysalt is this data generated by program? if so, can we only keep insert SQL in test case instead of these too large data

Reply:
@silverbullet233 by this SQL

```
    with T as (
  select
   days_add(years_add('1900-01-01', Y.delta), D.day) as start_day,
   days_sub(years_add('1900-12-31', Y.delta), D.day) as end_day,
   days_add(years_add('2000-01-01', Y.delta), D.day) as start_day2,
   days_sub(years_add('2000-12-31', Y.delta), D.day) as end_day2,
   M.mode
  from
  table(generate_series(0, 14, 1)) as D(day) join
  table(generate_series(0, 40, 1)) as Y(delta) join
  table(generate_series(0, 7, 1)) AS M(mode)```

But this SQL has to be executed on BE side, to get results of `week` function result on different inputs. And then use this output to do cross validation.